jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeSvcsApplication.java

```diff
@@ -22,9 +29,22 @@
 public class OdeSvcsApplication {
 
    private Logger logger = LoggerFactory.getLogger(this.getClass());
+   private static final int DEFAULT_NO_THREADS = 10;
+   private static final String DEFAULT_SCHEMA = "default";
 
-   public static void main(String[] args) {
+   public static void main(String[] args) throws MalformedObjectNameException, InterruptedException,
+         InstanceAlreadyExistsException, MBeanRegistrationException, NotCompliantMBeanException {
       SpringApplication.run(OdeSvcsApplication.class, args);
+      // Get the MBean server

```

Comment:
I don't see the point of the following lines of code. Looks like these are for a tutorial purpose and NOT relevant to our applications. Looks like this would be useful as a framework for exposing actual application metrics and not some constant fictitious thread count.